sensors/cmd/Dockerfile

```diff
@@ -4,7 +4,7 @@ RUN yum -y update && yum -y install ca-certificates openssh openssh-server opens
 # Argo Workflow CLI
 COPY assets/argo-linux-amd64 /usr/local/bin/argo
 
-RUN argo version
+RUN argo version || true
```

Comment:
Latest `argo` cli has an issue that it throws an error if no `kubeconfig` is provided.

Reply:
It looks like the issue has been fixed in `workflow`, let me create a PR to release a new CLI version and then we update it.

https://github.com/argoproj/argo/issues/3383

Reply:
It was merged 4 days ago, not in any release. 

It only impacts `docker build`, let's use `argo version || true` here for now to workaround the issue, and update it later with new CLI later.